srbhr/Resume-Matcher
# Added Annotated Text and Resume Parsing Functionality

## Pull Request Title
Added Annotated Text and Resume Parsing Functionality

## Related Issue
None

## Description
This pull request introduces several key changes to the application. The changes made include:

- Added the ability to display annotated text in the application.
- Implemented a feature that allows users to view extracted keywords.
- Enabled users to parse resumes within the app.

However, it's important to note that I'm encountering a couple of issues:
1. The application crashes frequently, specifically from backend port 8000, especially when there are changes in the backend code.
2. I have also made modifications to the implementation of the annotated text library.

Screenshots demonstrating these changes will be attached to the pull request.

## Type

## Proposed Changes
- Added annotated text feature.
- Introduced functionality to view extracted keywords.
- Enabled parsing of resumes.
- Made modifications to the annotated text library.

## Screenshots / Code Snippets (if applicable)
Screenshots will be provided with the pull request.

## How to Test
To test the changes in this pull request, follow these steps:

1. Clone this branch.
2. Run the application.
3. Check the functionality related to annotated text, keyword extraction, and resume parsing.
4. Pay attention to any crashes occurring from backend port 8000, especially after making changes to the backend code.

## Checklist

## Additional Information
Feel free to provide any additional information about the pull request, especially regarding the issues encountered and the changes made to the annotated text library implementation.
